client/src/pages/ProfileSetting/PaymentDetails/PaymentDetails.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC } from 'react';
+import { Elements } from '@stripe/react-stripe-js';
+import { loadStripe } from '@stripe/stripe-js';
+import PaymentForm from '../../../components/PaymentForm/PaymentForm';
+import PageContainer from '../../../components/PageContainer/PageContainer';
+
+const PaymentDetails: FC = (): JSX.Element => {
+  const stripePromise = loadStripe(process.env.REACT_APP_STRIPE_PUBLIC_KEY as string);
```

Comment:
I can't remember if you've asked me a question about this 😄  but it's a good practice to load stripe outside of a component's render. Please see image below:

![Capture](https://user-images.githubusercontent.com/75184965/127540368-445eb26a-5f1f-4516-b208-5db76c422dbf.PNG)

Reply:
You mentioned it and I actually read that and intended to do it, then didn't...
Done